[FIX] rename message_notification_light + black, isort, prettier

cvinh opened the pull request:
…ack, isort, prettier

lmignon committed:
Upgrade version from 8 -> 9

lmignon committed:
mail.notification has been removed. The notify is done directly on the res.partner and channel

Commit:
Add mail_optional_follower_notification module (#60)

* [ADD] Add mail_optional_follower_notifications module

* [CHG][mail_optional_follower_notifications] Readme.

* [IMP][mail_optional_follower_notifications] Code review.

* [CHG] Rename mail_optional_follower_notifications to mail_optional_follower_notification

* [IMP][mail_optional_follower_notification] Review

lmignon committed:
Tests: Replace XML_IDs no more available in 9.0

ThomasBinsfeld committed:
Argument auto_commit in send_mail

Commit:
Migration of mail_optional_follower_notification to 10.0

Commit:
[FIX] Bad version

sbidoul committed:
[FIX] pep8

Commit:
[FIX] mail_optional_follower_notification: Check force_partners_to_notify instead of notify_followers.
Without this commit, followers are NEVER notified when a message is directly posted on the tracker (without click to open the mail compose message form)

RobinetDenisAcsone committed:
[IMP] Hide the checkbox for the mass mailing in mail_optional_follower_notification and mail_optional_autofollow

pedrobaeza committed:
[FIX] README.rst syntax

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (5 of 5 strings)

Translation: social-10.0/social-10.0-mail_optional_follower_notification
Translate-URL: https://translation.odoo-community.org/projects/social-10-0/social-10-0-mail_optional_follower_notification/de/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (5 of 5 strings)

Translation: social-10.0/social-10.0-mail_optional_follower_notification
Translate-URL: https://translation.odoo-community.org/projects/social-10-0/social-10-0-mail_optional_follower_notification/pt/

astirpe committed:
[12.0][MIG] mail_optional_follower_notification

lmignon committed:
[IMP] mail_activity_board: Direct access to related model instance
Add a direct access from activity calendar and form views to the linked model instance.

Bedore this change, it was not possible to access to the model instance linked to an activity from the calendar view nor from the view opened by clicling on the activity into the calendar

[IMP] mail_activiy_board: Restrict related_model_instance selection to models supporting mail acitvities

Co-Authored-By: Miquel Raïch <miquel.raich@eficent.com>

lmignon committed:
[IMP] mail_optional_follower_notification: black, isort

lmignon committed:
[MIG] mail_optional_follower_notification: Migration to 13.0

enriquemartin committed:
[FIX] mail_restrict_follower_selection: res_model not always in context
When creating a record from a record from another model, the model is not in the context (`default_res_model` key). For example: creating an invoice from a sale order.

lmignon committed:
Update mail_optional_follower_notification/models/mail_thread.py

Co-authored-by: Isabelle RICHARD <isabelle.richard@smile.fr>

etobella committed:
[ADD] mail_preview_audio

Commit:
[IMP] mail_preview_audio: black, isort, prettier

Commit:
[13.0][MIG] mail_preview_audio

Commit:
[FIX] mail_preview_audio: force mail_preview_base js dependency

oca-travis committed:
[UPD] Update mail_preview_audio.pot

OCA-git-bot committed:
[UPD] README.rst

angelmoya committed:
[IMP] mail_restrict_follower_selection: Adding tests and improving pre-commit

Commit:
[MIG] mail_preview_audio: Migration to 14.0

Commit:
[IMP] pre-commit run -a

glitchov committed:
[13.0][FIX] mail_optional_follower_notification: change added span ref in view

etobella committed:
[ADD] mail_quoted_reply

ioans73 committed:
[IMP] mail_optional_follower_notification: black, isort, prettier

ioans73 committed:
[MIG] mail_optional_follower_notification: Migration to 14.0

ValentinVinagre committed:
ADD mail_show_follower copied from https://github.com/sygel-technology/sy-server-backend/tree/12.0/mail_show_follower

eLBati committed:
IMP mail_show_follower metadata and style
Display CC according to notification_type

oca-travis committed:
[UPD] Update mail_show_follower.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

eLBati committed:
FIX mail_show_follower: ValueError: Expected singleton: res.users

Commit:
[IMP] mail_tracking_mailgun: refactor to support modern webhooks

Before this patch, the module was designed after the [deprecated Mailgun webhooks][3]. However Mailgun had the [events API][2] which was quite different. Modern Mailgun has deprecated those webhooks and instead uses new ones that include the same payload as the events API, so you can reuse code.

However, this was incorrectly reusing the code inversely: trying to process the events API through the same code prepared for the deprecated webhooks.

Besides, both `failed` and `rejected` mailgun events were mapped to `error` state, but that was also wrong because [`mail_tracking` doesn't have an `error` state][1].

So the logic of the whole module is changed, adapting it to process the events API payload, both through controllers (prepared for the new webhooks) and manual updates that directly call the events API.

Also, `rejected` is now translated into `reject`, and `failed` is translated into `hard_bounce` or `soft_bounce` depending on the severity, as specified by [mailgun docs][2]. Also, `bounced` and `dropped` mailgun states are removed because they don't exist, and instead `failed` and `rejected` properly get their metadata.

Of course, to know the severity, now the method to obtain that info must change, it' can't be a simple dict anymore.

Added more parameters because for example modern Mailgun uses different keys for signing payload than for accessing the API. As there are so many parameters, configuration is now possible through `res.config.settings`. Go there to autoregister webhooks too.

Since the new webhooks are completely incompatible with the old supposedly-abstract webhooks controllers (that were never really that abstract), support for old webhooks is removed, and it will be removed in the future from `mail_tracking` directly. There is a migration script that attempts to unregister old webhooks and register new ones automatically.

[1]: https://github.com/OCA/social/blob/f73de421e28a43d018176f61725a3a59665f715d/mail_tracking/models/mail_tracking_event.py#L31-L42
[2]: https://documentation.mailgun.com/en/latest/api-events.html#event-types
[3]: https://documentation.mailgun.com/en/latest/api-webhooks-deprecated.html

cvinh committed:
[MIG] mail_footer_notified_partner: Migration to 14.0

dreispt committed:
[FIX] mail_allow_portal_internal_note: fix filter for messages to display

olgamarcocb committed:
[MIG] mail_restrict_follower_selection: Migration to 14.0

Kev-Roche committed:
[IMP] active / unactive mail adresses filter

Commit:
[MIG] website_mass_mailing_name: Migration to 14.0

cvinh commented:
cc @dreispt

cvinh commented:
cc @max3903

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (7 of 7 strings)

Translation: social-12.0/social-12.0-mail_show_follower
Translate-URL: https://translation.odoo-community.org/projects/social-12-0/social-12-0-mail_show_follower/fr/

rousseldenis committed:
[14.0][FIX] mail_outbound_static: Apply pre-commit

OCA-git-bot committed:
Merge PR #815 into 14.0

Signed-off-by dreispt

oca-travis committed:
[UPD] Update mail_partner_opt_out.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

olgamarcocb committed:
[FIX] mail_notification_custom_subject: Error modifying subtype_id

OCA-git-bot committed:
Merge PR #836 into 14.0

Signed-off-by etobella

oca-travis committed:
[UPD] Update mail_optional_follower_notification.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

jabelchi committed:
Added translation using Weblate (Catalan)

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (20 of 20 strings)

Translation: social-14.0/social-14.0-mail_autosubscribe
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_autosubscribe/ca/

OCA-git-bot committed:
Merge PR #830 into 14.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update website_mass_mailing_name.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

sebastienbeau committed:
[IMP] improve UI

olgamarcocb committed:
Translated using Weblate (Spanish)

Currently translated at 50.0% (2 of 4 strings)

Translation: social-14.0/social-14.0-mail_optional_follower_notification
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_optional_follower_notification/es/

ivantodorovich committed:
[FIX] mass_mailing_subscription_email: /unsubscribe_from_list link

OCA-git-bot committed:
Merge PR #844 into 14.0

Signed-off-by ivantodorovich

oca-travis committed:
[UPD] Update mass_mailing_subscription_email.pot

OCA-git-bot committed:
mass_mailing_subscription_email 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

ivantodorovich committed:
[FIX] mass_mailing_company_newsletter: missing permissions

To be able to create mailing.contact and its subscriptions, the
user needs to have at least the "Email Marketing/User" group.

This is not enough for most simple users, but they still need
to be able to set the partner's subscription from the partner form.

OCA-git-bot committed:
Merge PR #846 into 14.0

Signed-off-by ivantodorovich

OCA-git-bot committed:
mass_mailing_company_newsletter 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

jcoux committed:
[14.0][FIX] mass_mailing_subscription_email: fix unsubscribe

OCA-git-bot committed:
Merge PR #849 into 14.0

Signed-off-by ivantodorovich

OCA-git-bot committed:
mass_mailing_subscription_email 14.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #850 into 14.0

Signed-off-by pedrobaeza

oca-travis committed:
[UPD] Update mail_tracking_mailgun.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_tracking_mailgun
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_tracking_mailgun/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (6 of 6 strings)

Translation: social-14.0/social-14.0-mail_debrand
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_debrand/nl/

rafaelbn committed:
[12.0][mail_show_follower] Update version

As commented in https://github.com/OCA/social/pull/788 the PR should be merged manually but we should update the version of the module as a minor update

cc @ValentinVinagre @eLBati @MiquelRForgeFlow @HaraldPanten

rafaelbn committed:
Update mail_show_follower/__manifest__.py

Co-authored-by: Miquel Raïch <miquel.raich@eficent.com>

MateuGForgeFlow committed:
[14.0][IMP] mail_debrand: Remove Powered by Odoo

dreispt commented:
Hi @cvinh this PR has merge conflicts.
It seems that it is repeating changes already done in the main PR.
Can you make this with a commit with only your corrections please?

Shide committed:
[IMP] mail_show_follower: black, isort, prettier

Shide committed:
[MIG] mail_show_follower: Migration to 15.0

[REF] Compact comment

OCA-git-bot committed:
Merge PR #834 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update mail_outbound_static.pot

OCA-git-bot committed:
mail_outbound_static 14.0.2.0.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #826 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update mail_drop_target.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
mail_drop_target 14.0.1.0.1

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #822 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update mail_filter_adressee_by_contact.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

pedrobaeza committed:
[FIX] mail_attach_existing_attachment_account: Resilient tests on multi-company

Doing a bulk search on accounts without company filter can lead to
selecting an account of other company, and thus, making the test to
fail.

We restrict the account search to the current company.

OCA-git-bot committed:
[UPD] addons table in README.md

Shide committed:
[ADD] Customize notification appearance

Added settings to customize notification and translations to important parts of the message.

[FIX] Security fixes and simplified customization

Field `show_followers_partner_format` formatted with %(param)s
Removed CSS customizations

[FIX] README options

[FIX] Index.html configure

[FIX] Several changes

- Removed unnecesary properties on res_config_settings
- Markup safe on partner_format
- Use email_domain_extract instead own one

tafaRU committed:
[MIG] mail_show_follower: Backporting from 15.0

tafaRU committed:
[IMP] mail_show_follower: black, isort, prettier

Kev-Roche committed:
[IMP] differenciate users and partners in mail filter

grindtildeath committed:
[FIX] mass_mailing_partner: Fix assignation of company name

Defining a res.company on a partner should not set the company_name
on a mailing contact as the mailing contact is not linked to a
res.company.
The company_name should instead be set according to the commercial
partner of the partner being modified if it's parent_id is changed.

Commit:
[FIX] email_template_qweb: get translations working

Translations were broken, things hav changed in 14.0 and we need to call
`_classify_per_lang` method to get the right lang, and use it to render
the content.

OCA-git-bot committed:
Merge PR #867 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update mail_show_follower.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_show_follower
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_show_follower/

moylop260 committed:
[REF] mass_mailing_partner: Use width inheritance and prevents buttons from wrapping to fit the page

`oe_inline` makes the `button.width = auto !important;`
 - https://github.com/OCA/social/blob/f5d73f90527367e5036a95d10cb911ab076748b9/mass_mailing_partner/views/res_partner_view.xml#L25

Causing the `button.oe_inline` to have a different size than the others (depending on your text):
 - https://github.com/odoo/odoo/blob/a9395e2460f8cd22a6b6c59b0ad3d5ae67e2847f/addons/web/static/src/scss/form_view.scss#L41

Since the `auto !important;` breaks the width inheritance and prevents buttons from wrapping to fit the page width

Co-authored-by: Manuel Gómez <manuel@vauxoo.com>

OCA-git-bot committed:
Merge PR #869 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update base_search_mail_content.pot

oca-travis committed:
[UPD] Update mail_activity_done.pot

oca-travis committed:
[UPD] Update mail_activity_team.pot

oca-travis committed:
[UPD] Update mail_filter_adressee_by_contact.pot

oca-travis committed:
[UPD] Update mail_tracking.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mail_filter_adressee_by_contact 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_activity_team
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_activity_team/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_activity_done
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_activity_done/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_tracking
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_tracking/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-base_search_mail_content
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-base_search_mail_content/

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #876 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
mass_mailing_partner 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

jonasbuchholz committed:
[MIG] mail_quoted_reply: Migration to 14.0

MiquelRForgeFlow committed:
[UPD] Eficent -> ForgeFlow

OCA-git-bot committed:
Merge PR #885 into 14.0

Signed-off-by rafaelbn

oca-travis committed:
[UPD] Update mail_activity_board.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mail_activity_board 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_activity_board
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_activity_board/

Shide committed:
[FIX] Add recipients of the all related messages on the notification

When using composer, doesn't include all recipients of the message on the notification.

OCA-git-bot committed:
Merge PR #858 into 14.0

Signed-off-by rafaelbn

OCA-git-bot committed:
mail_debrand 14.0.2.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #886 into 14.0

Signed-off-by rafaelbn

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mail_debrand 14.0.2.2.0

OCA-git-bot committed:
mail_activity_team 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

Shide committed:
[FIX] mail_show_follower: Drop compatibility with HTML

HTML fields displayed on res_config_settings uses too much space.
`show_followers_message_sent_to` and `show_followers_message_response_warning` are now Text fields.

Shide committed:
[FIX] Make onchange private

Make private onchange_show_followers_message_preview

Co-authored-by: Jairo Llopis <Yajo@users.noreply.github.com>

OCA-git-bot committed:
Merge PR #893 into 14.0

Signed-off-by Yajo

oca-travis committed:
[UPD] Update mail_show_follower.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mail_show_follower 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_show_follower
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_show_follower/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (13 of 13 strings)

Translation: social-14.0/social-14.0-mass_mailing_subscription_email
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mass_mailing_subscription_email/pt/

OCA-git-bot committed:
Merge PR #874 into 14.0

Signed-off-by etobella

oca-travis committed:
[UPD] Update mail_preview_audio.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #872 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
email_template_qweb 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

JoanMForgeFlow committed:
[FIX] mail_activity_team. smart button archive

OCA-git-bot committed:
Merge PR #810 into 14.0

Signed-off-by dreispt

oca-travis committed:
[UPD] Update mail_allow_portal_internal_note.pot

OCA-git-bot committed:
mail_allow_portal_internal_note 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #823 into 14.0

Signed-off-by etobella

oca-travis committed:
[UPD] Update mail_restrict_follower_selection.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_restrict_follower_selection
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_restrict_follower_selection/

grindtildeath committed:
mail_tracking_mass_mailing: Set traces in error according to tracking

If an exception is caught by ir.mail_server.send_email in mail_tracking module,
the mail.tracking.email record will appear in error but the related mailing
trace would still appear as sent because mail.mail._postprocess_sent_message is
called without any failure_type in mail.mail._send in the mail module (as Exception
is not raised after being caught in mail_tracking module).

Since _postprocess_sent_message method not only sets the mailing.trace state in
mass_mailing module but can also delete the mail.mail records in mail module,
we need to ensure the mailing.trace is written accordingly to the tracking here,
and avoid having the mass_mailing module set a 'sent' status if we had an exception,
hence the usage of a context key to ignore possible writes.

OCA-git-bot committed:
Merge PR #908 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
mail_tracking_mass_mailing 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

miguel-S73 committed:
[14.0][FIX] mail_debrand: remove_href_odoo function

- FIX bug when split email message by `to_keep` parameter. When the html code is split, it is broken. As a result of this, when it use `fromstring`,
it try to fix this by closing tags.
Example about fixed functionality:

message:
`
<table>
   <tr>
      <td>
         {to_keep}
      </td>
   </tr>
</table>
`
Split messsage inside `remove_href_odoo` function:

part[0]

`
<table>
   <tr>
      <td>
`
--------
part[1]
`
      </td>
   </tr>
</table>
`
-----------------
Result when join by `to_keep`

`
<table>
<tr>
<td>
{to_keep}
</td>
</tr>
</table>
`

OCA-git-bot committed:
Merge PR #911 into 14.0

Signed-off-by rafaelbn

OCA-git-bot committed:
mail_debrand 14.0.2.2.1

OCA-git-bot committed:
[UPD] addons table in README.md

ivantodorovich committed:
[ADD] mail_layout_force

LoisRForgeFlow committed:
[IMP] black, isort, prettier

LoisRForgeFlow committed:
[MIG] mail_quoted_reply: migration to version 13.0

Commit:
[UPD] Update mail_quoted_reply.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #921 into 14.0

Signed-off-by etobella

oca-travis committed:
[UPD] Update mail_quoted_reply.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

Kev-Roche committed:
Added translation using Weblate (French)

Kev-Roche committed:
Translated using Weblate (French)

Currently translated at 80.0% (8 of 10 strings)

Translation: social-14.0/social-14.0-mail_filter_adressee_by_contact
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_filter_adressee_by_contact/fr/

jabelchi committed:
Added translation using Weblate (Catalan)

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (8 of 8 strings)

Translation: social-14.0/social-14.0-mail_quoted_reply
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_quoted_reply/ca/

jabelchi committed:
Added translation using Weblate (Spanish)

jabelchi committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: social-14.0/social-14.0-mail_quoted_reply
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_quoted_reply/es/

OCA-git-bot committed:
Merge PR #916 into 14.0

Signed-off-by simahawk

oca-travis committed:
[UPD] Update mail_layout_force.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] icon.png

Commit:
[IMP] mail_tracking: remove legacy hooks

OCA-git-bot committed:
Merge PR #942 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
mail_tracking 14.0.2.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

etobella committed:
Updated from template

OCA-git-bot committed:
Merge PR #955 into 14.0

Signed-off-by etobella

jarmokortetjarvi committed:
Added translation using Weblate (Finnish)

jarmokortetjarvi committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (19 of 19 strings)

Translation: social-14.0/social-14.0-mail_show_follower
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_show_follower/fi/

LoisRForgeFlow committed:
[FIX] mail_quoted_reply: avoid access right errors when reading action.

OCA-git-bot committed:
Merge PR #977 into 14.0

Signed-off-by etobella

OCA-git-bot committed:
mail_quoted_reply 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

Hardik-OSI committed:
[14.0][FIX]Portal User Dont Have Access for Commercial Partner

OCA-git-bot committed:
Merge PR #978 into 14.0

Signed-off-by dreispt

OCA-git-bot committed:
mail_allow_portal_internal_note 14.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #735 into 14.0

Signed-off-by rafaelbn

Commit:
[UPD] Update mail_optional_autofollow.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
mail_optional_autofollow 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_optional_autofollow
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_optional_autofollow/

ivantodorovich committed:
[FIX] mass_mailing_subscription_email: Avoid repetition emails if opt_out hasn't really changed

OCA-git-bot committed:
Merge PR #983 into 14.0

Signed-off-by ivantodorovich

OCA-git-bot committed:
mass_mailing_subscription_email 14.0.1.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #899 into 14.0

Signed-off-by etobella

Commit:
[UPD] Update mail_activity_team.pot

OCA-git-bot committed:
mail_activity_team 14.0.1.1.1

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: social-14.0/social-14.0-mail_activity_team
Translate-URL: https://translation.odoo-community.org/projects/social-14-0/social-14-0-mail_activity_team/

cvinh committed:
Merge pull request #2 from OCA/14.0

Rebase
